README.md

```diff
@@ -28,7 +28,13 @@ ICICLE is a CUDA implementation of general functions widely used in ZKP. ICICLE
 
 ## Build and usage
 
-> NOTE: [NVCC] is a prerequisite for building.
+
+### Prerequisites
+
+- [NVCC]
+- cmake 3.24.0 and above (though 3.22.1 has been tested and works as well)
```

Comment:
I think 3.18 is the lowest supported cmake version.
Its probably better to have an example of using the CLI variable in the `Testing the CUDA code` section below for versions < 3.24 instead of making 3.24 a prereq.